odc/geo/geobox.py

```diff
@@ -422,6 +423,55 @@ def qr2sample(
             n, padding=padding, with_edges=with_edges, offset=offset
         )
 
+    # pylint: disable=redefined-builtin, anomalous-backslash-in-string
+    def explore(
+        self,
+        map: Optional[Any] = None,
+        grid_lines: bool = True,
+        tiles: Any = "OpenStreetMap",
+        attr: Optional[str] = None,
+        map_kwds: Optional[Dict[str, Any]] = None,
+        **kwargs: Any,
+    ) -> Any:
+        """
+        Plot a visual representation of the GeoBox's extent on an
+        interactive :py:mod:`folium` leaflet map.
+
+        :param map:
+            An optional existing :py:mod:`folium` map object to plot into.
+            By default, a new map object will be created.
+        :param: grid_lines:
+            Whether to plot pixel edge aligned grid lines over the GeoBox
+            extent.
+        :param tiles:
+            Map tileset to use for the map basemap. Supports any option
+            supported by :py:mod:`folium`, including "OpenStreetMap",
+            "CartoDB positron", "CartoDB dark_matter" or a custom XYZ URL.
+        :param attr:
+            Map tile attribution; only required if passing custom tile URL.
+        :param map_kwds:
+            Additional keyword arguments to pass to ``folium.Map()``.
+        :param \**kwargs:
+            Additional keyword arguments to pass to ``folium.GeoJson``.
```

Comment:
```suggestion
            Additional keyword arguments to pass to :py:class:`folium.GeoJson`.
```

Reply:
With that and after exposing folium docs in here:

https://github.com/opendatacube/odc-geo/blob/37ec5a185e0333eefa06ea766bff32239a55008c/docs/conf.py#L83-L92

we might get click through to the folium docs hopefully, which are here:
https://python-visualization.github.io/folium/latest/

Reply:
Have made this change in a few places, but it doesn't look like the click-through is working: https://65c0897265175afa7ad50c31--odc-geo-docs.netlify.app/_api/odc.geo.geobox.geobox.explore